app/api/feedbacks.py

```diff
@@ -113,11 +114,12 @@ def before_update_object(self, feedback, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        if feedback.session_id:
+        if feedback and feedback.session_id:
             session = Session.query.filter_by(id=feedback.session_id).first()
-            if not has_access('is_organizer', event_id=session.event_id):
-                raise ForbiddenException({'source': ''},
-                                         "Event organizer access required")
+            if session:
+                if not has_access('is_organizer', event_id=session.event_id):
```

Comment:
Use and

Reply:
@iamareebjamal `and` ? Which logic should I and it with?

Reply:
Oops. Understood. Updating it right away